packages/core/src/components/slider/coreSlider.tsx

```diff
@@ -124,8 +125,11 @@ export abstract class CoreSlider<P extends ICoreSliderProps> extends AbstractCom
     }
 
     private maybeRenderAxis() {
-        const { max, min, labelStepSize } = this.props;
-        if (this.props.renderLabel === false) { return undefined; }
+        const { max, min } = this.props;
+        const labelStepSize = this.props.labelStepSize || (max - min);
```

Comment:
are we still safe if `labelStepSize === 0 && max === min`?
also, i thought we're avoiding boolean coercion?